patches/common/chromium/.patches.yaml

```diff
@@ -469,3 +469,9 @@ patches:
       * Fixes a main_application_delegate SDK change
       * Fixes a non-null SDK change in a net unittest.
     This is needed for Electron to compile with XCode 10.0.
+-
+  author: Shelley Vohr <shelley.vohr@gmail.com>
+  file: backport_baaa9de.patch
+  description: |
+    Backports baaa9de, a patch which resolves
```

Comment:
Please leave a link to the Chromium change.
Either a commit or a review.

Reply:
@miniak this may need to be closed, i've been tracking the issue and we have all the changes ostensibly made to fix this in 69 already

Reply:
@codebytere we don't have the fix, refer https://github.com/electron/electron/pull/14865#issuecomment-428534594